discovery/gossiper.go

```diff
@@ -107,13 +107,14 @@ type Config struct {
 }
 
 // AuthenticatedGossiper is a subsystem which is responsible for receiving
-// announcements validate them and apply the changes to router, syncing
-// lightning network with newly connected nodes, broadcasting announcements
-// after validation, negotiating the channel announcement proofs exchange and
-// handling the premature announcements. All outgoing announcements are
-// expected to be properly signed as dictated in BOLT#7, additionally, all
-// incoming message are expected to be well formed and signed. Invalid messages
-// will be rejected by this struct.
+// announcements, validating them and applying the changes to router, syncing
```

Comment:
Scanning this diff fragment, I can't see any change other than the line-wrapping being modified (to a smaller col limit?). Can you revert this section of the diff if there're no material change? Thanks!